Modul OOP/engine.py

```diff
@@ -0,0 +1,39 @@
+import models
+import settings
+import exceptions
+
+
+def get_player_name() -> str:
+    name = ''
+    while name == '':
+        name = (input("ENTER YOUR NAME: ")).strip()
+    return name
+
+
+def play() -> None:
+    level = settings.INITIAL_ENEMY_LEVEL
+    name = get_player_name()
+    scores = open('scores.txt', 'w')
+    scores.write(name)
+    player = models.Player(name)
+    enemy = models.Enemy(level)
```

Comment:
Це як раз випадок, коли ліпше не створювати зайві змінні.
Тобто ти просто імпортував значення, прив'язав до нього ще одну зміну, використав один раз і забув про неї. Ліпше напряму закидати потрібне значення.
```suggestion
    enemy = models.Enemy(settings.INITIAL_ENEMY_LEVEL)
```